web/src/main/kotlin/no/nav/su/se/bakover/web/routes/vedtak/VedtakJson.kt

```diff
@@ -92,6 +93,8 @@ internal fun Avslagsvedtak.toJson(): VedtakJson = VedtakJson(
     periode = periode.toJson(),
     type = VedtakTypeJson.AVSLAG.toString(),
     dokumenttilstand = this.dokumenttilstand.toJson(),
+    // avslagsvedtak er per tidspunkt det eneste vedtaket som kan starte en ny form for behandling
+    kanStarteNyBehandling = true,
```

Comment:
Dette er vel for enkelt? Hva skjer hvis en tom sak har:
1) søknad
2) avslag
3) innvilget over avslaget
4) Da kan man fremdeles trykke på opprett ny søknadsbehandling fra avslag. Kanskje det er greit 🤷 Man får ikke lov til å ha overlappende stønadsperiode uansett

Reply:
dette har vi vel ikke tatt stilling til enda